lib/src/store.rs

```diff
@@ -229,7 +230,7 @@ impl Store {
         path: &RepoPath,
         id: &ConflictId,
     ) -> BackendResult<MergedTreeValue> {
-        let backend_conflict = block_on(self.backend.read_conflict(path, id))?;
+        let backend_conflict = self.backend.read_conflict(path, id).block_on()?;
```

Comment:
> I don't think we need to worry about reading legacy conflicts
> asynchronously - async is really only useful for Google's backend
> right now, and we don't use the legacy format at Google.

Maybe we can change the backend `read_conflict()` non-async and document why? We can even rename it to `read_legacy_conflict()`.

I'm not against introducing `pollster` dependency, but it would be confusing if we had two ways of blocking async calls without lengthy comment.

Reply:
Ah, of course! That's a much simpler solution. Done.